backend/test/session.test.js

```diff
@@ -70,6 +70,6 @@ describe('Testing /api/session endpoint ', () => {
       .expect('Content-Type', /json/)
       .expect(200);
 
-    expect(data.body.token).toStrictEqual({ token: { user_id: 'dummy_id' } });
+    expect(data.body).toStrictEqual({ token: { user_id: 'dummy_id' } });
```

Comment:
@marcianszlovak this is not supposed to be like this, the `data.body` should be
```js
{
  token: 'jwtToken',
  user_id: 'dummy_id',
  isAdmin: true
}
```
so that test should fail